src/navigate-pop/using-workspaces.md

```diff
@@ -0,0 +1,30 @@
+# Workspaces
+
+Manage multiple workflows and projects with Workspaces. Show and access Workspaces by clicking on the icon with multiple windows in the dock, clicking `Workspaces` in the top left corner of your screen, or by pressing `SUPER` + `D`.
+
+![Show Workspaces](/images/workspaces/show-workspaces.png)
+
+The open windows for a given Workspace will fan across the screen when that workspace is selected.
```

Comment:
You capitalized Workspace the first time you said it and not the second time. I would recommend only capitalizing the word when referring to the name of the menu, and not capitalizing every time you use the word "workspace" (similar to the Applications menu vs. the applications the reader's using.) But whichever you go for, it should be consistent.